content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -137,20 +153,55 @@ use, see `man 7 user_namespaces`.
 
 ## Set up a node to support user namespaces
 
-It is recommended that the host's files and host's processes use UIDs/GIDs in
-the range of 0-65535.
-
-The kubelet will assign UIDs/GIDs higher than that to pods. Therefore, to
-guarantee as much isolation as possible, the UIDs/GIDs used by the host's files
-and host's processes should be in the range 0-65535.
+By default, the kubelet assumes host's files and host's processes use UIDs/GIDs
+in the range of 0-65535. This is the case in most Linux distributions with the
+default installation. The kubelet will assign UIDs/GIDs higher than that to
+pods, so there is no overlap between UID/GIDs used by host's files and processes
+and the ones used by pods.
 
 Note that this recommendation is important to mitigate the impact of CVEs like
 [CVE-2021-25741][CVE-2021-25741], where a pod can potentially read arbitrary
-files in the hosts. If the UIDs/GIDs of the pod and the host don't overlap, it
+files in the host. If the UIDs/GIDs of the pod and the host don't overlap, it
 is limited what a pod would be able to do: the pod UID/GID won't match the
 host's file owner/group.
 
+The kubelet can use a custom range for pods too. To configure a custom range,
+the node needs to have:
+
+ * A user `kubelet` in the system
+ * The binary `getsubids` installed (part of the [shadow-utils][shadow-utils] package)
```

Comment:
We don't know what the package name is, because we don't know what OS they are using.